src/elemeno_ai_sdk/ml/features/feature_table.py

```diff
@@ -1,208 +1,48 @@
-import datetime
-from typing import Any, Dict, List
+import os
+from typing import Any, Dict, List, Optional
 
-import feast
-import feast.types
-from feast.data_format import JsonFormat
+from elemeno_ai_sdk.ml.features.schema import FeatureTableSchema
+from elemeno_ai_sdk.ml.features.utils import get_feature_server_url_from_api_key
+from elemeno_ai_sdk.ml.mlhub_client import MLHubRemote
 
-from elemeno_ai_sdk.ml.features.types import FeatureType
 
-
-class FeatureTable:
+class FeatureTable(MLHubRemote):
     """A FeatureTable is the object that is used to define feature tables on Elemeno feature store.
 
     If you're looking to create a new feature table or read data look at ingest_schema of the class FeatureStore.
     """
 
-    def __init__(
-        self,
-        name: str,
-        feature_store: feast.FeatureStore = None,
-        entities: List[feast.Entity] = None,
-        features: List[feast.Feature] = None,
-        ttl_duration_weeks=52,
-        is_streaming=False,
-        online=False,
-        event_column: str = "event_timestamp",
-        created_column: str = "created_timestamp",
-    ):
-        self.name = name
-        self._entities = [] if entities is None else entities
-        self._features = [] if features is None else features
-        self._feast_elm = feature_store
-        self._duration = ttl_duration_weeks
-        self._online = online
-        self.is_streaming = is_streaming
-        self._evt_col = event_column
-        self._created_col = created_column
-        self._original_schema = None
-        self._table_schema = []
-
-    @property
-    def entities(self) -> List[feast.Entity]:
-        return self._entities
-
-    @property
-    def evt_col(self):
-        return self._evt_col
-
-    @property
-    def created_col(self):
-        return self._created_col
+    def __init__(self, schema_path: str, remote_server: Optional[str] = None):
+        if remote_server is None:
+            api_key = os.getenv("MLHUB_API_KEY")
```

Comment:
Sugiro usar o os.environ se não for usar um default. Melhor que a falta de chave quebre no init da classe